<launch>
  <node pkg="autolander"
    type="autolander_node"
    name="lander_node"
    output="screen">

    <!-- Topic repeaters -->
    <param name="throttleInputTopic" type="string" value="mavros/thrust_IN"/>
    <param name="attitudeInputTopic" type="string" value="mavros/att_IN"/>

    <param name="twMax" value="1.7"/>
    <param name="wifiTopic" type="string" value="10801"/>
    <param name="disarmThresh" value="0.5"/>


  </node>
</launch>